server/session.go

```diff
@@ -255,12 +257,24 @@ func (s *Session) isCluster() bool {
 	return s.isProxy() || s.isMultiplex()
 }
 
+func (s *Session) unlockTerminating() {
+	if s == nil {
+		return
+	}
+	s.terminatingLock.Unlock()
```

Comment:
I think it's safe to repalce `s.unlockTerminating()` with `s.terminatingLock.Unlock()`.

Reply:
Done.